lib/cit-v1.jsonld

```diff
@@ -14,6 +14,46 @@
         "meta": {"@id": "https://w3id.org/cit#meta", "@type": "https://w3id.org/security#multibase"},
         "payload": {"@id": "https://w3id.org/cit#payload", "@type": "https://w3id.org/security#multibase"}
       }
+    },
+    "Ed25519Signature2020": {
+      "@id": "https://w3id.org/security#Ed25519Signature2020",
+      "@context": {
+        "@version": 1.1,
```

Comment:
Remove `@version` everywhere, not necessary.

Reply:
@dlongley removed, thanks.